scripts/jenkins/cloud/ansible/roles/deploy_caasp_terraform/defaults/main.yml

```diff
@@ -0,0 +1,78 @@
+#
+# (c) Copyright 2020 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+ca_path: "/etc/pki/trust/anchors/ca-bundle.pem"
+caasp_deployment_path: "/home/sles/caasp/deployment"
+caasp_openstack_path: "{{ caasp_deployment_path }}/openstack"
+caasp_osrc: "openstack_caasp.osrc"
+caasp_user: "sles"
+cpiautotfvars_path: "{{ caasp_openstack_path }}/cpi.auto.tfvars"
+dns_server_ip1: "10.84.2.20"
+dns_server_ip2: "10.84.2.21"
+image_url: "http://ibs-mirror.prv.suse.net/install/SLE-15-SP1-JeOS-QU2/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU2.qcow2"
+image_path: "{{ soc_home }}/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU2.qcow2"
+image_name: "SLE-15-SP1-JeOS-QU2"
+timestamp: ''
+internal_net: "{{'caasp-int-net' + '-' + timestamp }}"
+internal_subnet: "{{'caasp-int-subnet' + '-' + timestamp}}"
+keyname: "caaspmanagementkey"
+lb_tf_path: "{{ caasp_openstack_path }}/load-balancer.tf"
+master_size: "m1.medium"
```

Comment:
The ECP large flavor gives a 80GB disk and inside the VM it's partitioned so the root fs is only 51GB. The m1.medium requires 40GB, so can only fit one master on a ECP VM as disk over subscription is disabled. But the master is only using ~5GB.  Maybe create new flavor with the same parameters as m1.medium but with only 10GB for disk. Same goes for workers.

Reply:
Updated to use a custom flavor.